tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -118,68 +117,1144 @@ - (void)dealloc {
         ASSERT_EQ_MSG(xamlElement.Get(), nullptr, "Unexpectedly able to reacquire a strong reference to the backing Xaml element.");
     }
 
-    TEST_METHOD(UIButton_TitleColorChanged) {
-        __block auto uxEvent = UXEvent::CreateManual();
+    TEST_METHOD(UIButton_CreateButtonWithDefaultInit) {
+        StrongId<UIButtonWithControlsViewController> buttonVC;
+        buttonVC.attach([[UIButtonWithControlsViewController alloc] init]);
+        UXTestAPI::ViewControllerPresenter testHelper(buttonVC);
+
+        UIButton* buttonToTest = [buttonVC defaultButton];
+
+        WXCControl* xamlControl = rt_dynamic_cast([WXCControl class], [buttonToTest xamlElement]);
+        ASSERT_OBJCNE(xamlControl, nil);
+
+        // Extract UIButton.titleLabel control to verify its visual state
+        WXFrameworkElement* titleLabelXamlElement = [buttonToTest.titleLabel xamlElement];
+        ASSERT_OBJCNE(titleLabelXamlElement, nil);
+
+        WXCTextBlock* textBlock = rt_dynamic_cast([WXCTextBlock class], titleLabelXamlElement);
+        ASSERT_OBJCNE(textBlock, nil);
+
+        // Background color
+        dispatch_sync(dispatch_get_main_queue(), ^{
+            WUXMSolidColorBrush* solidBackgroundBrush = rt_dynamic_cast([WUXMSolidColorBrush class], xamlControl.background);
+            UIColor* expectedBackgroundColor = [UIColor colorWithRed:1.0f green:1.0f blue:1.0f alpha:0.0f];
+            EXPECT_TRUE(UXTestAPI::IsRGBAEqual(solidBackgroundBrush, expectedBackgroundColor));
+        });
+
+        // Title color
+        __block auto uxEvent = UXEvent::CreateAuto();
         __block auto xamlSubscriber = std::make_shared<XamlEventSubscription>();
 
-        UIButtonWithControlsViewController* buttonVC = [[UIButtonWithControlsViewController alloc] init];
+        dispatch_async(dispatch_get_main_queue(), ^{
+            // Register RAII event subscription handler
+            xamlSubscriber->Set(textBlock, [WXCTextBlock foregroundProperty], ^(WXDependencyObject* sender, WXDependencyProperty* dp) {
+                WUXMSolidColorBrush* solidBrush = rt_dynamic_cast([WUXMSolidColorBrush class], [sender getValue:dp]);
+
+                // Validation
+                if (UXTestAPI::IsRGBAEqual(solidBrush, [UIColor whiteColor])) {
```

Comment:
Are you're sure it's not UIButtonTypeSystem? That one is blue by default. In my testing, UIButtonTypeCustom (and default) is white. IslandWood also sets its default to white.

---
In reply to: [100680288](https://github.com/Microsoft/WinObjC/pull/1947#discussion_r100680288) [](ancestors = 100680288)